Makefile

```diff
@@ -177,19 +187,91 @@ scan:
 		--ignore-unfixed \
 		./
 
-print-%:
-	@echo -n $($*)
+.PHONY: build-utils-image
```

Comment:
where is this used?

Reply:
its not just like build-local-artifact-mirror-image is not